package/cell_modem_daemon/src/at_command_utils.c

```diff
@@ -274,10 +263,9 @@ const char *at_serial_port_command_result(at_serial_port_command_t *at_command)
  *  99 - not known or not detectable
  */
 static const s8 signal_to_dbm_lookup[] = {
-  -113, // zero case
-  -111, -109, -107, -105, -103, -101, -99, -97, -95, -93,  // 1 - 30 cases
-  -91,  -89,  -87,  -85,  -83,  -81,  -79,  -77, -75, -73,
-  -71, -69,  -67,  -65,  -63,  -61,  -59,  -57,  -55, -53,
+  -113,                                                   // zero case
+  -111, -109, -107, -105, -103, -101, -99, -97, -95, -93, // 1 - 30 cases
+  -91,  -89,  -87,  -85,  -83,  -81,  -79, -77, -75, -73, -71, -69, -67, -65, -63, -61, -59, -57, -55, -53,
```

Comment:
I believe this is the result of the formatter attempting to pack arguments to the new 120 column max, while also respecting trailing comments. Also we may have to reject double slash comments at some point, I've heard that's a thing in MISRA et. all